crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -124,7 +124,7 @@ pub fn calculate_bounds(
 }
 
 pub fn update_frusta<T: Component + CameraProjection + Send + Sync + 'static>(
-    mut views: Query<(&GlobalTransform, &T, &mut Frustum)>,
+    mut views: Query<(&GlobalTransform, Raw<T>, &mut Frustum)>,
```

Comment:
The need for this sort of change makes me worried: I don't love that users will have to reason about this when trying to write generic systems.

Reply:
Maybe, but it's a fairly mindless fix - just use the Raw and RawMut wrappers for directly generic components. I think ergonomic wins for the more common cases outweigh this slight loss here.